### PR TITLE
Branch to practice GitHub flow

### PR DESCRIPTION
**Pull request by kkolade**
On creating the branch, the following changes were made
- H1 was added to the index.html file
- The H1 was styled in the style.css file
- test.md file was added and the file was added to .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,104 +1,2 @@
...
-
-# FuseBox cache
-.fusebox/
-
-# DynamoDB Local files
-.dynamodb/
-
-# TernJS port file
-.tern-port
+#Test File
```
Comment (ichala) [inline .gitignore:1]: - [ ] Good Job creating the .gitignore file 👏 , as part of the requirements  kindly add the `node_modules` directory inside of it as well

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @kkolade 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Pr Description✔️ 
- Meaningful commits ✔️


**_Note: Good Job so far, your commit message is clear and meaningful but kindly make sure in your next projects, The commit message should be written in the imperative tense and capitalized example: Add styling to the header, Add .gitignore file, etc ..._**


 ## Required Changes ♻️

- [x] 🛑 Good Job Adding the correct linters configurations, but unfortunately they are not working, the test should run when you open a pull request as shown in [this guide](https://github.com/microverseinc/linters-config/tree/master/html-css), the current issue is that you mistyped the name of the folder `workflows`, it should be `workflows` and not `workflow`, kindly fix it and make sure that the tests are running 


- [x] 🛑  **_Check The comments below_**



## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
The correction from the Code Reviewer has been implemented

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @kkolade , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good use of github flow :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

 ## Required Changes ♻️

- [ ] Good job so far :+1:  But I noticed that your linter:cry: Files stylelintrc.json, .hintrc are missing. Please follow this [Linters HTML and CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) and make your linters work as requested by the previous reviewer. Linters are very important because they will help to become a better developer by fixing code quickly and having your code syntax-correct. :+1: 

Screenshot from 2022-10-25 09-17-56
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
The required linter files were added and the header selector in stle.css was corrected based on an error generated by the linter

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @kkolade, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Githu flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**